pkg/install/install_test.go

```diff
@@ -84,28 +73,16 @@ func TestDeployARMTemplate(t *testing.T) {
 					Wait(ctx, resourceGroup, deploymentName).
 					Return(wait.ErrWaitTimeout)
 			},
-			wantErr: wait.ErrWaitTimeout,
-		},
-		{
-			name: "Resource quota exceeded error",
-			mocks: func(dc *mock_features.MockDeploymentsClient) {
-				dc.EXPECT().
-					CreateOrUpdateAndWait(ctx, resourceGroup, deploymentName, deployment).
-					Return(quotaErr)
-			},
-			wantErr: cloudQuotaErr,
+			wantErr: "400: DeploymentFailed: : ARM deployment failed. Details: 0: : : timed out waiting for the condition",
```

Comment:
I'm not a big fan of the way `func (err *CloudError) Error() string` writes `: ` even if a field is empty.